test/e2e/common_test.go

```diff
@@ -599,6 +599,24 @@ func (p *PodmanTestIntegration) CreateSeccompJson(in []byte) (string, error) {
 	return jsonFile, nil
 }
 
+func SkipIfRootlessCgroupsV1(reason string) {
+	if reason == "" {
+		panic("SkiptIfRootlessCgroupsV1 must specify a reason to skip")
+	}
+	if os.Geteuid() != 0 && !CGROUPSV2 {
+		Skip("[rootless]: " + reason)
+	}
+}
+
+func SkipIfRootless(reason string) {
+	if reason == "" {
```

Comment:
How about `if len(reason) < 5` instead? (And, will `len()` handle nil?) When doing this type of check, I sometimes worry about people bypassing it with `" "` or `"x"` or `"foo"`. (Never deliberately, it's just that sometimes we write code in a hurry and try to get it passing with the best intentions to fix it later). Ibid below.

Reply:
Fixed